Sources/PrimerSDK/Classes/Data Models/ClientToken.swift

```diff
@@ -28,6 +28,10 @@ struct DecodedJWTToken: Codable {
     var qrCode: String?
     var accountNumber: String?
     
+    // iPay88
+    var backendCallbackUrl: String?
+    var primerTransactionId: String?
```

Comment:
Is there any plan to split this file into multiple files? It seems that having only one file for all possibilities of the client token is a bit hard to follow 🤔

Reply:
Yeah, there is once the `clientToken` is refactored.